test/ui-test/lightspeedUiTest.ts

```diff
@@ -49,15 +49,25 @@ export function lightspeedUIAssetsTest(): void {
       // this.retries(3);
       // this.timeout(20000); // even 18s failed
 
-      const loginButton = lightspeedServiceSection
-        .findWelcomeContent()
-        .then(async (val) => {
-          return (await val?.getButtons()) as WelcomeContentButton[];
-        });
+      const welcomeContent =
+        await lightspeedServiceSection.findWelcomeContent();
+      expect(welcomeContent).not.undefined;
+      const loginButton = await welcomeContent?.findElement(
+        By.xpath("//a[.//span/text()='Connect']")
+      );
+      expect(loginButton).not.undefined;
+
+      // **** ORIGINAL CODE ****
```

Comment:
Do we need the old code commented out? If not this can be removed and instead add some comments to explain the new approach.

Reply:
I fully agree here, we should almost never have code commented. We have git history for that.